ArduCopter/GCS_Mavlink.cpp

```diff
@@ -762,7 +762,7 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_int_packet(const mavlink_command_i
 }
 
 #if HAL_MOUNT_ENABLED
-MAV_RESULT GCS_MAVLINK_Copter::handle_command_mount(const mavlink_command_long_t &packet, const mavlink_message_t &msg)
+MAV_RESULT GCS_MAVLINK_Copter::handle_command_long_mount(const mavlink_command_long_t &packet, const mavlink_message_t &msg)
```

Comment:
This doesn't look right.  We should be accepting *anything* as *just* command_long.